kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/wires/PropertiesUi.ui.xml

```diff
@@ -91,6 +101,27 @@
 					</b:Panel>
 				</b:ModalBody>
 			</b:Modal>
+			<b:Modal closable="true" fade="true" dataBackdrop="STATIC"
+			dataKeyboard="true" b:id="new-channel-modal" ui:field="newChannelModal">
+			<b:ModalHeader/>
+			<b:ModalBody>
+				<b:Form>
+					<b:FormGroup addStyleNames="{style.channel-name-form}">
+						<b:FormLabel ui:field="newChannelNameLabel"/>
+						<g:FlowPanel>
+							<b:TextBox ui:field="newChannelNameInput" b:id="channelName" allowBlank="false"
+								autoComplete="false"/>
+						</g:FlowPanel>
+						<b:FormLabel addStyleNames="{style.channel-name-validation-label}" ui:field="newChannelNameError"/>
+					</b:FormGroup>
+				</b:Form>
+			</b:ModalBody>
+			<b:ModalFooter>
+                <b:Button b:id="btn-create-comp-cancel" icon="REMOVE" type="PRIMARY" dataDismiss="MODAL" ui:field="btnCancelCreatingNewChannel"/>
```

Comment:
kindly change the ID of the button to something like `btn-create-channel-cancel`